src/less/components/links.less

```diff
@@ -80,9 +145,4 @@ a.action-link {
         color: @text-gray;
         padding-left: 0.5rem;
     }
-
-    // Temporary solution until DG-323 (https://payexjira.atlassian.net/secure/RapidBoard.jspa?rapidView=80&projectKey=DG&modal=detail&selectedIssue=DG-323) is solved [AW].
-    > * {
-        margin-right: 0.5rem;
-    }
 }
```

Comment:
Focus state on a regular Link is not correct

Reply:
Can't see the problem. Pls tell me in huddle